nemo/collections/asr/data/audio_to_text_lhotse_prompted.py

```diff
@@ -61,22 +62,43 @@ class PromptedAudioToTextLhotseDataset(torch.utils.data.Dataset):
     Tokenized utterances will be extended with special prompt tokens according to ``prompt_format_fn`` logic.
     We support cuts with multiple supervision segments -- their tokenized texts will be concatenated before we add the prompt tokens.
     This is useful, for example, in code-switched scenarios where each segment is spoken in a different language.
+
+    Chunking:
+    If `enable_parallel_chunking` is True, each audio sample is split into optimally sized chunks
```

Comment:
sorry I think its better to keep it just as 
```suggestion
    If `enable_chunking` is True, each audio sample is split into optimally sized chunks
```